src/natives.cpp

```diff
@@ -8,9 +8,10 @@ The code here acts as the translation between AMX data types and native types.
 */
 
 #include "natives.hpp"
+#include "pawn-natives/NativeFunc.hpp"
```

Comment:
I should have mentioned this on the last review (sorry!) but why is there a mix of `""` and `<>` include methods for pawn-natives? I say stick with `<>` for portability.